src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,36 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        Car prado = new Car();
+        prado.setModel("Prado");
+        prado.setManufacturer(new Manufacturer(2L, "Toyota", "Japan"));
```

Comment:
Why do you put 'id' into new Manufacturer and Driver? You can see in Dao implementation that this 'id' is skipped. Database generates it's own id.
![image](https://user-images.githubusercontent.com/117547096/219878984-147dfddb-2c8e-46d3-b9be-69a4a649c930.png)

It's better to create new manufacturer using ManufacturerService and then put it into car. Same for new Driver